platform/src/main/resources/example.anchor-config.yaml

```diff
@@ -83,7 +83,7 @@ sep12:
 
 sep24:
   enabled: true
-  interactiveUrl: http://localhost:8081/sep24/interactive
+  interactive_url: http://localhost:8083/sep24/interactive
```

Comment:
Its hard to understand whats being changed for each configuration file.. why are there so many 😅

Reply:
I'm pretty sure most of them are unused, but just in case I changed it everywhere

Reply:
Gleb has a good suggestion to further simplified the configuration yaml file. I have reduced the number of files once but it kind of growing back as the project progresses. 

I had created a [story ](https://stellarorg.atlassian.net/browse/ANCHOR-120) to track.